core/src/test/java/org/mskcc/cbio/portal/util/TestGenePanelUtil.java

```diff
@@ -0,0 +1,217 @@
+package org.mskcc.cbio.portal.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.mskcc.cbio.portal.util.GenePanelUtil.*;
+
+import org.junit.Test;
+import org.mskcc.cbio.portal.model.CanonicalGene;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Properties;
+import java.util.Set;
+
+public class TestGenePanelUtil {
+    Properties p1;
+    Properties p2;
+    Properties p3;
+    Properties p4;
+    Properties p5;
+    Properties p6;
+    Set<CanonicalGene> g1;
+    Set<CanonicalGene> g2;
+    Set<CanonicalGene> g3;
+    Set<CanonicalGene> g4;
+    Set<CanonicalGene> g5;
+    Set<CanonicalGene> g6;
+
+    CanonicalGene otor = makeGene(13321, 56914, "OTOR", makeAliases("FDP", "MIAL1"));
```

Comment:
Make these class variables, remove the gene sets, create gene sets as local variables in each test case.